lightning/src/routing/scoring.rs

```diff
@@ -798,6 +797,122 @@ impl<G: Deref<Target = NetworkGraph>, T: Time> Score for ProbabilisticScorerUsin
 	}
 }
 
+mod approx {
+	const BITS: u32 = 64;
+	const HIGHEST_BIT: u32 = BITS - 1;
+	const LOWER_BITS: u32 = 4;
+	const LOWER_BITS_BOUND: u64 = 1 << LOWER_BITS;
+	const LOWER_BITMASK: u64 = (1 << LOWER_BITS) - 1;
+
+	/// Look-up table for `log10(x) * 1000` where row `i` is used for each `x` having `i` as the
```

Comment:
Ugh, can we make this `log10(x) * 1024` instead so that we don't have to divide by 1000 in the penalty calculation (and can instead just divide by 1024, which compiles down to bit-shift).